t/base/protocols.t

```diff
@@ -18,3 +18,5 @@ ok(!$ua->protocols_allowed(),           '! $ua->protocols_allowed');
 
 $ua->protocols_forbidden(undef);
 ok(!$ua->protocols_forbidden(),         '$ua->protocols_forbidden(undef)');
+
+done_testing();
```

Comment:
done_testing requires Test::More 0.88, so either the prereq needs to be raised, or (more preferably) we stick with `plan`s.